src/Raven.Server/Storage/Schema/Updates/Documents/60000/From50002.cs

```diff
@@ -0,0 +1,132 @@
+using static Raven.Server.Documents.Schemas.Conflicts;
+using static Raven.Server.Documents.Schemas.Counters;
+using static Raven.Server.Documents.Schemas.DeletedRanges;
+using static Raven.Server.Documents.Schemas.Documents;
+using static Raven.Server.Documents.Schemas.Revisions;
+using static Raven.Server.Documents.Schemas.TimeSeries;
+using static Raven.Server.Documents.Schemas.Tombstones;
+
+namespace Raven.Server.Storage.Schema.Updates.Documents
+{
+    public class From50002 : ISchemaUpdate
+    {
+        public int From => 50_002;
+
+        public int To => 60_000;
+        
+        public SchemaUpgrader.StorageType StorageType => SchemaUpgrader.StorageType.Documents;
+
+        internal static int NumberOfItemsToMigrateInSingleTransaction = 10_000;
+
+        private long _processedSinceLastCommit;
+
+        public bool Update(UpdateStep step)
+        {
+            InsertIndexValuesFor(step, DocsSchemaBase60, DocsBucketAndEtagSlice,
+                fixedSizeIndex: DocsSchemaBase60.FixedSizeIndexes[AllDocsEtagsSlice],
+                etagPosition: (int)DocumentsTable.Etag);
+
+            InsertIndexValuesFor(step, TombstonesSchemaBase60, TombstonesBucketAndEtagSlice,
+                fixedSizeIndex: TombstonesSchemaBase60.FixedSizeIndexes[AllTombstonesEtagsSlice],
+                etagPosition: (int)TombstoneTable.Etag);
+
+            InsertIndexValuesFor(step, RevisionsSchemaBase60, RevisionsBucketAndEtagSlice,
+                fixedSizeIndex: RevisionsSchemaBase60.FixedSizeIndexes[AllRevisionsEtagsSlice],
+                etagPosition: (int)RevisionsTable.Etag);
+
+            InsertIndexValuesFor(step, TimeSeriesSchemaBase60, TimeSeriesBucketAndEtagSlice,
+                fixedSizeIndex: TimeSeriesSchemaBase60.FixedSizeIndexes[AllTimeSeriesEtagSlice],
+                etagPosition: (int)TimeSeriesTable.Etag);
+
+            InsertIndexValuesFor(step, DeleteRangesSchemaBase60, DeletedRangesBucketAndEtagSlice,
+                fixedSizeIndex: DeleteRangesSchemaBase60.FixedSizeIndexes[AllDeletedRangesEtagSlice],
+                etagPosition: (int)DeletedRangeTable.Etag);
+
+            InsertIndexValuesFor(step, CountersSchemaBase60, CountersBucketAndEtagSlice,
+                fixedSizeIndex: CountersSchemaBase60.FixedSizeIndexes[AllCountersEtagSlice],
+                etagPosition: (int)CountersTable.Etag);
+
+            InsertIndexValuesFor(step, ConflictsSchemaBase60, ConflictsBucketAndEtagSlice,
+                fixedSizeIndex: ConflictsSchemaBase60.FixedSizeIndexes[AllConflictedDocsEtagsSlice],
+                etagPosition: (int)ConflictsTable.Etag);
+
+            InsertIndexValuesFor(step, AttachmentsSchemaBase60, AttachmentsBucketAndEtagSlice,
+                fixedSizeIndex: AttachmentsSchemaBase60.FixedSizeIndexes[AttachmentsEtagSlice],
+                etagPosition: (int)AttachmentsTable.Etag,
+                tableName: AttachmentsMetadataSlice.ToString());
```

Comment:
Why are you passing this only here?

Reply:
Attachments are handled differently.. 
I'm not sure why, but they have no indexes that are marked as global
and I wasn't able to read them using `table = new Table(schema, transaction)` overload
see also here : https://github.com/ravendb/ravendb/blob/v5.3/src/Raven.Server/Documents/AttachmentsStorage.cs#L116